styles/mixins.scss

```diff
@@ -34,9 +34,37 @@
     }
 }
 
-@mixin position($value) {
-    left: $value;
-    right: $value;
-    top: $value;
-    bottom: $value;
+//
+// Sandstone Font Applicator
+//
+@mixin sand-font(
+    $family: variables.$sand-font-family,
+	$nlfamily: variables.$sand-non-latin-font-family,
```

Comment:
It looks better to name it more readable. We know nl means non-latin now, but maybe it is unclear for someone in the future. :)